### ISSUE TITLE
Gear color doesn't apply onto alt appearance

### ISSUE THREAD

**Issue by grunbag** [WarEmu/WarBugs#9649]
I made some test with different gear pieces, and it appear that some gear dye doesn't apply on alt appearance.

let's see drengi boots : base color blue 
![skori_013](https://cloud.githubusercontent.com/assets/24207191/26045402/55105c9a-394a-11e7-9950-f92aaa153806.jpg)

i want to apply this alt appearance, base color brown
![skori_014](https://cloud.githubusercontent.com/assets/24207191/26045409/5b6dca28-394a-11e7-859d-2e76926aa1f8.jpg)

so it supposed to take the blue drengi boots color but it doesn't and keep the brown color or dire wold boots
![skori_015](https://cloud.githubusercontent.com/assets/24207191/26045455/a1230556-394a-11e7-9205-828d8c397605.jpg)

note : the boots can be dye properly

some other pieces works fine like annihilator helm : base color black and red
![skori_016](https://cloud.githubusercontent.com/assets/24207191/26045466/b3816580-394a-11e7-807c-0316dca741f4.jpg)

i want to apply this helm : metho's hardhat 
![skori_017](https://cloud.githubusercontent.com/assets/24207191/26045474/bda368ec-394a-11e7-87e6-667c89d5548d.jpg)

so black and red from anni correctly apply on it
![skori_018](https://cloud.githubusercontent.com/assets/24207191/26045509/ec4e17aa-394a-11e7-8db6-3f881d296ee2.jpg)

same test with duelist set : base color blue, light blue and red
![skori_019](https://cloud.githubusercontent.com/assets/24207191/26045549/11846b3c-394b-11e7-9c8d-0b76b79d1ab2.jpg)

i want to apply the lair set : base color brown, red, purple
![skori_020](https://cloud.githubusercontent.com/assets/24207191/26045555/1742534a-394b-11e7-9578-10d855eb5a09.jpg)

it supposed to take blue, light blue and red from duelist set

but it doesn't : 
![skori_021](https://cloud.githubusercontent.com/assets/24207191/26045561/1d9cbe92-394b-11e7-8ce8-2994426e8945.jpg)

**Comment by Luth-RoR** [comment]
Do the alternate appearance items can be died with a second coloer when equipped normally?

**Comment by grunbag** [comment]
Yes they can , but it appear that several items can't take color. This is related to some figleaf files that can't be edited now.
